packages/sit-onyx/src/components/OnyxDataGrid/features/all.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./sorting/types";
+
+export { useDataGridSorting as useSorting } from "./sorting/sorting";
```

Comment:
Whats the purpose of this re-export?

Reply:
In the entry file `index.ts` I export everything from this file scoped to `DataGridFeatures`.
Two reasons:
- The features (will) have a lot of types and I don't want to clutter the main `sit-onyx` scope too much.
- I want the user to have a concise and structured way of using features, so by putting this under `DataGridFeatures` we can achieve this.